frontend/src/scenes/project/CreateProjectModal.tsx

```diff
@@ -39,25 +39,25 @@ export function CreateProjectModal({
         <LemonModal
             title={currentOrganization ? `Create a project in ${currentOrganization.name}` : 'Create a project'}
             description={
-                <p>
-                    Most companies will want 3 projects:
-                    <br />
-                    1. Local Development
-                    <br />
-                    2. Staging
-                    <br />
-                    3. Production
-                    <br />
-                    <br />
-                    <strong>Tip:</strong> we recommend using the same project for both your website and app to track
-                    across them. You can always apply a filter to focus on just one.{' '}
-                    <Link to="https://posthog.com/manual/organizations-and-projects#projects" target="_blank">
-                        Learn more here.
-                    </Link>
-                    <br />
-                    <br />
-                    <strong>Bonus tip:</strong> you can rename your "Default Project" to "Production".
-                </p>
+                <>
+                    <p>For most companies, we recommend using 3 projects:</p>
+                    <ol className={'mt-2'}>
+                        <li>Local Development</li>
+                        <li>Staging</li>
+                        <li>Production</li>
+                    </ol>
+                    <p>
```

Comment:
I definitely think this is useful, but thinking about this a bit more, presentation as a 123 list just isn't particularly slick here. I propose we just explain that this is a common use case in a quick sentence. This would be annoying to do with PR edit suggestions, so I made a PR on top of this one, take a look: #13403 😅 

<img width="499" alt="1" src="https://user-images.githubusercontent.com/4550621/208453667-13d53137-199f-4ff3-9cb3-142c283e7f2a.png"> |